frontend/src/components/pages/CampOverview/CampersTable/CampersTable.tsx

```diff
@@ -211,12 +216,7 @@ const CampersTable = ({
                     padding="0px"
                     maxWidth="32px"
                   >
-                    <IconButton
-                      aria-label="Mark as active button"
-                      icon={<FaEllipsisV />}
-                      variant=""
-                      // onClick={() => console.log("3 dot button pressed!")}
-                    />
```

Comment:
Hmm not sure why the linter isn't catching this one, but are there unused `IconButton` and `FaEllipsisV` imports now?

Reply:
yeah that's strange, I ran lint fix but i guess it didn't remove them